installation_script.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+main_script_name="translate.py"
+main_script_path="$PWD/src/$main_script_name"
+python_path=$1
+python_path_size=${#python_path}
+echo Path to translator script is $main_script_path
+
+default_python=1
+if [[ $python_path_size -eq 0 ]]; then echo "Using default system python"; else echo "Using python interpreter at: '$python_path'"; default_python=0; fi
+
+if [[ $default_python -eq 0 ]] # if python path is provided check if path is correct (contains python)
+then
+    version=$($python_path -V 2>&1 | grep -Po '(?<=Python )(.+)') # get python version from python path
+    if [[ -z "$version" ]]
+    then
+        echo "Error, incorrect python path!"
+        exit 1 # exit script if python path is incorrect
+    fi
+fi
+
+read -p "Enter desired alias for translator: " main_translate_alias
+read -p "Enter alternative alias for translator(leave blank if not needed): " second_translate_alias
+
+echo  >> ~/.bashrc # add empty line at the end of file
```

Comment:
A request of taking into consideration alternative run commands [rc] files.
.zshrc (newest manjaro)
.kshrc
.bashrc

Especially .zshrc file is desired to be supported. The script can use a specific name depending on which file has been detected and if none of the custom ones, than .bashrc should be use as a default (many users keep both .bashrc and another installed rc files in their home directory). A message that informs the use of the location of the alias would also be beneficial to avoid a potential confusion